### PR TITLE
Feature/elbil24

### PR DESCRIPTION
**Pull request by jimoe**
#### Please tick a box ###
- [ ] **feature** _(A new feature_)
- [ ] **fix** _(A bug fix_)
- [ ] **refactor** _(A code change that neither fixes a bug nor adds a feature_)
- [ ] **docs** _(Documentation only changes_)
- [ ] **performance** _(A code change that improves performance_)
- [ ] **style** _(Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)_)
- [ ] **build** _(Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)_)
- [ ] **ci** _(Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)_)
- [ ] **test** _(Adding missing tests or correcting existing tests_)

#### If you're adding a feature, is it documented in a storybook story?
- [ ] Yes
- [ ] No
- [ ] Not adding a new feature

#### Are the components you're working on reusable between brands?
- [ ] Yes _(Using variables that are defined in the default theme)_
- [ ] No _(I hard coded some values)_
- [ ] Not working on any components

#### If you're creating markup, did you add proper semantics? 
- [ ] I added [ARIA attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)
- [ ] I added microdata from [Schema.org](http://schema.org/)
- [ ] Semantics are derived from HTML
- [ ] Not applicable

_(Did you do a CR and see that there is something that we should check for each PR, that are not on the list, please [update this document](https://github.com/dbmedialab/shiny/edit/master/.github/PULL_REQUEST_TEMPLATE.md))_

### CODE REVIEW

File: src/atoms/Elbil24Logo.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+		fill: ${props => (props.pro ?getColor('secondary') : getColor('primary'))}
+	}
+`;
+
+/* eslint-disable max-len */
+const Elbil24Logo = props => (
+	<Svg
+		role="img"
+		aria-labelledby="title"
+		id="Elbil24Logo"
```
Comment (misund) [inline src/atoms/Elbil24Logo.js]: HTML restricts the same ID to be used only once per page. Remove it.

Add a `<title>Elbil24</title>` inside the Svg, comparable to `alt="Elbil24"` on img tags.

File: src/atoms/Elbil24ProButton.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	}
+
+	& .st1 {
+		fill: ${getColor('secondary')}
+	}
+`;
+
+/* eslint-disable max-len */
+const Elbil24ProButton = props => (
+	<Svg
```
Comment (misund) [inline src/atoms/Elbil24ProButton.js]: Remove ids (from the paths as well), add <title>

File: src/themes/elbil24/index.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+import colors from './colors';
+
+const variables = {
+	mainFont: "'Open Sans', helvetica, arial, sans-serif",
+	headingsFont: "'Open Sans', helvetica, arial, sans-serif",
+};
+
+const global = `
+	@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,700,800');
```
Comment (misund) [inline src/themes/elbil24/index.js]: Do you use all these weights? uiWeight, uiWeightBold and uiWeightSemibold is defined in theme.variables - but that's only 3 different weights. Here there are four.

File: stories/link-bars-complete-examples/Elbil24Story.js [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+};
+
+// const TopNavigation = styled(HorizontalLinkBar)`
+const TopNavigation = styled.div`
+	background: ${getColor('primary')};
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	width: 100%;
+	height: 145px;
```
Comment (misund) [inline stories/link-bars-complete-examples/Elbil24Story.js]: rems are better

File: stories/link-bars-complete-examples/Elbil24Story.js [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+`;
+
+const ProLink = styled(Link)`
+	position: absolute;
+	font-size: 12px;
+	text-decoration: none;
+	color: #fff;
+	text-align: right;
+	margin-top: -138px;
+	margin-left: -55px;
```
Comment (misund) [inline stories/link-bars-complete-examples/Elbil24Story.js]: rems are better

File: stories/link-bars-complete-examples/Elbil24Story.js [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+	flex-direction: column;
+	align-items: center;
+	padding-top: 2rem;
+	width: 18rem;
+`;
+
+const Slogan = styled(Paragraph)`
+	color: ${getColor('white')};
+	font-style: italic;
+	font-size: 13px;
```
Comment (misund) [inline stories/link-bars-complete-examples/Elbil24Story.js]: rems are better

File: stories/link-bars-complete-examples/Elbil24Story.js (hunk 4 above)
Comment (jimoe) [reply stories/link-bars-complete-examples/Elbil24Story.js]: Will fix things like that when we have final html/css

File: src/atoms/Elbil24ProButton.js (hunk 2 above)
Comment (jimoe) [reply src/atoms/Elbil24ProButton.js]: Ups.  Forgot to remove those :(

File: src/atoms/Elbil24ProButton.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+			d="M67.9565,30.7211 C68.6065,30.0191 68.9345,29.0811 68.9345,27.9051 L68.9345,20.3961 C68.9345,19.2211 68.6065,18.2831 67.9565,17.5801 C67.3065,16.8781 66.4425,16.5281 65.3705,16.5281 C64.3225,16.5281 63.4715,16.8781 62.8205,17.5801 C62.1715,18.2831 61.8445,19.2211 61.8445,20.3961 L61.8445,27.9051 C61.8445,29.0811 62.1715,30.0191 62.8205,30.7211 C63.4715,31.4241 64.3225,31.7751 65.3705,31.7751 C66.4425,31.7751 67.3065,31.4241 67.9565,30.7211 M59.7555,36.6601 C58.1355,35.8041 56.8785,34.6051 55.9835,33.0581 C55.0895,31.5131 54.6435,29.7201 54.6435,27.6771 L54.6435,20.4721 C54.6435,18.4801 55.0895,16.7191 55.9835,15.1861 C56.8785,13.6531 58.1355,12.4671 59.7555,11.6231 C61.3785,10.7801 63.2505,10.3581 65.3705,10.3581 C67.5155,10.3581 69.3985,10.7801 71.0215,11.6231 C72.6425,12.4671 73.8985,13.6531 74.7935,15.1861 C75.6875,16.7191 76.1335,18.4801 76.1335,20.4721 L76.1335,27.6771 C76.1335,29.7201 75.6875,31.5131 74.7935,33.0581 C73.8985,34.6051 72.6425,35.8041 71.0215,36.6601 C69.3985,37.5151 67.5155,37.9431 65.3705,37.9431 C63.2505,37.9431 61.3785,37.5151 59.7555,36.6601"
+		/>
+		<polygon id="Fill-9" fill="#000000" points="0.0004 39.9384 0.0004 58.0244 15.5254 47.4344" />
```
Comment (misund) [inline src/atoms/Elbil24ProButton.js]: Just one more ID left

**Pull request merged**